doc/cprover-manual/contracts-history-variables.md

```diff
@@ -13,8 +13,9 @@ _ensures_ clause.
 
 ### Parameters
 
-`__CPROVER_old` takes a single argument, which is the identifier corresponding to
-a parameter of the function. For now, only scalar or pointer types are supported.
+`__CPROVER_old` takes a single argument, which is the identifier
+corresponding to a parameter of the function. For now, only scalars,
+pointers, and struct members are supported.
```

Comment:
The `if` condition that was updated below only allows dereference and struct members. Do we need to have some other condition for simple scalars?

Reply:
Good catch, we needed to add `ID_symbol`. I also added a new regression test to cover this case.